### PR TITLE
sash/update jquery/react

### PR DESCRIPTION
**Pull request by sashcy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,8 +131,8 @@
     "moment": "2.19.4",
     "promise-polyfill": "6.1.0",
     "prop-types": "15.6.1",
-    "react": "16.6.3",
-    "react-dom": "16.6.3",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
```
Comment (negarn) [inline package.json]: we want to use exact versions, so for all of these please remove `^` from the version

File: package.json [hunk 2]
```diff
@@ -84,15 +84,15 @@
...
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
-    "load-grunt-config": "1.0.1",
+    "load-grunt-config": "^1.0.2",
     "load-grunt-tasks": "3.5.2",
     "mocha": "5.2.0",
     "mock-local-storage": "1.1.7",
     "node-gettext": "2.0.0",
-    "node-sass": "4.11.0",
+    "node-sass": "^4.12.0",
```
Comment (negarn) [inline package.json]: @sashcy this and load-grunt-config are still using `^` 😅

File: package.json [hunk 3]
```diff
@@ -84,15 +84,15 @@
...
     "postcss-scss": "0.4.1",
     "react-render-html": "0.6.0",
-    "react-svg-loader": "^2.1.0",
+    "react-svg-loader": "^3.0.1",
```
Comment (negarn) [inline package.json]: also this

**Comment by negarn** [comment]
@sashcy could you resolve the conflicts?

**Comment by sashcy** [comment]
> @sashcy could you resolve the conflicts?

Conflicts resolved

File: package.json [hunk 4]
```diff
@@ -118,15 +118,23 @@
...
+    "mobx-react": "5.4.2",
+    "mobx-utils": "5.4.0",
     "moment": "2.19.4",
     "promise-polyfill": "6.1.0",
     "prop-types": "15.6.1",
-    "react": "16.6.3",
-    "react-dom": "16.6.3",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-pose": "4.0.8",
```
Comment (cakasuma) [inline package.json]: afaik, we won't use react-pose, react-router-dom, and react-transition-group on binary static, that would be in deriv app

File: package.json [hunk 5]
```diff
@@ -118,15 +118,23 @@
     "davidshimjs-qrcodejs": "0.0.2",
     "event-source-polyfill": "0.0.9",
     "highstock-release": "5.0.14",
-    "jquery": "3.2.1",
+    "iscroll": "5.2.0",
+    "jquery": "3.4.1",
     "jquery.scrollto": "2.1.2",
     "js-cookie": "2.2.0",
     "kinetic": "5.2.0",
+    "lodash.debounce": "4.0.8",
```
Comment (cakasuma) [inline package.json]: is this necessary?

File: package.json [hunk 6]
```diff
@@ -118,15 +118,23 @@
     "davidshimjs-qrcodejs": "0.0.2",
     "event-source-polyfill": "0.0.9",
     "highstock-release": "5.0.14",
-    "jquery": "3.2.1",
+    "iscroll": "5.2.0",
```
Comment (cakasuma) [inline package.json]: may i know where does this used?, or is this actually sub dependency?

**Comment by negarn** [comment]
@sashcy I think during resolving the conflict you added the packages which were removed by negar/remove_app_2, please remove them again

**Comment by sashcy** [comment]
> @sashcy I think during resolving the conflict you added the packages which were removed by negar/remove_app_2, please remove them again

sorry, cleaned-up

File: package.json (hunk 5 above)
Comment (sashcy) [reply package.json]: cleaned

File: package.json (hunk 4 above)
Comment (sashcy) [reply package.json]: cleaned

File: package.json [hunk 7]
```diff
@@ -83,15 +83,15 @@
     "grunt-webpack": "3.1.3",
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
-    "load-grunt-config": "1.0.1",
+    "load-grunt-config": "1.0.2",
```
Comment (cakasuma) [inline package.json]: do we need to update load-grunt-config and node-sass?

Comment (sashcy) [reply package.json]: npm shows security warning suggesting update

Comment (sashcy) [reply package.json]: npm 6.4.1

**Pull request merged**